wcomponents-core/src/main/java/com/github/bordertech/wcomponents/container/WrongStepAjaxInterceptor.java

```diff
@@ -170,7 +175,7 @@ private void handleRenderRedirect(final PrintWriter writer) {
 		writer.print("<ui:ajaxtarget id=\"" + triggerId + "\" action=\"replace\">");
 
 		// Redirect URL
-		writer.print("<ui:redirect url=\"" + redirectUrl + "\" />");
+		writer.print(String.format("<%1$s url=\"%2$s\"></%1$s>", TAG_REDIRECT, redirectUrl));
```

Comment:
Nice! 
No need to overuse the argument index, you should be able to only use it where you need it, so it could look something like this:

```java
writer.print(String.format("<%s url=\"%s\"></%1$s>", TAG_REDIRECT, redirectUrl));
```